bob-common/src/metrics/exporter/send.rs

```diff
@@ -1,13 +1,13 @@
 use log::{debug, trace};
 use std::collections::HashMap;
-use std::time::Duration;
-use tokio::sync::mpsc::Receiver;
-use tokio::time::{interval, timeout};
+use std::time::{Duration, Instant};
+use tokio::sync::mpsc::{channel, Receiver, Sender};
+use tokio::time::timeout;
 
 use super::retry_socket::RetrySocket;
 use super::{Metric, MetricInner, MetricKey, MetricValue, TimeStamp};
 
-const METRICS_RECV_TIMEOUT: Duration = Duration::from_millis(100);
+const TCP_CHANNEL_SIZE: usize = 1024;
```

Comment:
I suggest to name it differently, like `TCP_SENDER_QUEUE_SIZE` or `TCP_SENDER_BUFFER_SIZE` or smth like that